Future/R/macro.R

```diff
@@ -0,0 +1,60 @@
+#' @title Amount of macronutrients in product
+#'
+#' @description Function for calculating amount of macronutrients in product
+#'
+#' @param product string, name of used product
+#' @param weight numeric, weight of used product
+#'
+#' @examples
+#' macronutrients("Kawior", 50)
+#'
+#' @export
+macronutrients <- function(product, weight){
+
+  assertthat::assert_that(assertthat::is.string(product),
```

Comment:
Please use one assert method - in separate task fix all aserts FUT-19

Reply:
ok